docs/Guides/Testing.md

```diff
@@ -1,12 +1,20 @@
-<h1 align="center">Fastify</h1>
+<h1 style="text-align: center;">Fastify</h1>
 
-## Testing
+# Testing
+<a id="testing"></a>
 
 Testing is one of the most important parts of developing an application. Fastify
 is very flexible when it comes to testing and is compatible with most testing
 frameworks (such as [Tap](https://www.npmjs.com/package/tap), which is used in
 the examples below).
 
+**Table of contents**
```

Comment:
Not needed as the site will generate a ToC from the markdown headers: https://fastify.dev/docs/latest/Guides/Testing